catroid/src/test/java/org/catrobat/catroid/test/content/actions/RepeatActionTest.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.test.content.actions
+
+import org.catrobat.catroid.content.Script
+import org.catrobat.catroid.content.Sprite
+import org.catrobat.catroid.content.StartScript
+import org.catrobat.catroid.content.bricks.ChangeYByNBrick
+import org.catrobat.catroid.content.bricks.RepeatBrick
+import org.catrobat.catroid.content.eventids.EventId
+import org.catrobat.catroid.formulaeditor.Formula
+import org.catrobat.catroid.test.utils.Reflection
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class RepeatActionTest {
+    private var sprite: Sprite? = null
+    private var script: Script? = null
+    private var repeatBrick: RepeatBrick? = null
+    private var changeYByNBrick: ChangeYByNBrick? = null
+    private var repeatTimesFormula: Formula? = null
+
```

Comment:
please change all the var's to lateinit  and not nullable e.g.  "private lateinit var sprite: Sprite"
with this you don't need to force the variables to asume that they are not null

.!! without a null check is just hoping that there can't occur a nullpointer exception